src/App.js

```diff
@@ -1,16 +1,45 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
+
+import Heading from './Heading';
+import TouristInfoCards from './TouristInfoCards';
+import Bookings from './Bookings';
+import Restaurant from './Restaurant';
+import Footer from './Footer';
+import './App.css';
+import './grid.css';
+
 
-import Bookings from "./Bookings";
-import Restaurant from "./Restaurant";
-import "./App.css";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">CYF Hotel</header>
+        <Heading />
         <Bookings />
+       <div className='row'>
+        <TouristInfoCards
```

Comment:
`TouristInfoCards` (with an "s" at the end) implies that this component shows multiple cards, but in fact it just shows 1.

I would either rename it to be singular or (even better I think) rename them to be singular and also extract out showing all three cards to another component named `TouristInfoCards`